samples/subsys/fs/spi_sd/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _spi_sd:
+
+FAT Rd/Wr access Sample Application
+###################################
+
+Overview
+********
+
+This sample app demonstrates the write and read speed performances
+of the FAT filesystem API.
+The system mounts an SDHC card connected over a SPI bus or to an on-chip
+SDHC controller.
+
+Requirements
+************
+
+This project requires a SDHC or microSD card formatted with FAT filesystem.
+See the :ref:`sdhc_api` documentation for Zephyr implementation details.
+
+Building and Running
+********************
+
+This sample can be built for a variety of boards, including ``ci-shield``.
```

Comment:
I don't believe we have in-tree support for the `ci_shield`? Perhaps reference another board/shield?

Reply:
this  is the purpose of PR #26741

Reply:
Yes. I understand that, but we should not reference a shield not yet in the tree.

Reply:
removing any ref to the board